winml/lib/Api.Image/TensorToVideoFrameConverter.cpp

```diff
@@ -561,7 +561,7 @@ void TensorToVideoFrameConverter::ConvertGPUTensorToSoftwareBitmap(
   }
 
   uint32_t tensorElementSize = tensorDesc.dataType == kImageTensorDataTypeFloat32 ? 4 : 2;
-  uint32_t singleVideoFramebufferSize = static_cast<uint32_t>(tensorDesc.sizes[1] * tensorDesc.sizes[2] * tensorDesc.sizes[3] * tensorElementSize);
+  uint64_t singleVideoFramebufferSize = static_cast<uint64_t>(tensorDesc.sizes[1] * tensorDesc.sizes[2] * tensorDesc.sizes[3] * tensorElementSize);
```

Comment:
Should cast the first one first. 

static_cast<uint64_t>(tensorDesc.sizes[1]) * tensorDesc.sizes[2] * tensorDesc.sizes[3] * tensorElementSize;

It could reduce the chance of having integer overflow.